program-test/src/programs.rs

```diff
@@ -61,6 +61,11 @@ static CORE_BPF_PROGRAMS: &[(Pubkey, Option<Pubkey>, &[u8])] = &[
         None,
         include_bytes!("programs/core_bpf_feature_gate-0.0.1.so"),
     ),
+    (
+        solana_sdk_ids::stake::ID,
+        None,
+        include_bytes!("programs/core_bpf_stake-1.0.0.so"),
+    ),
```

Comment:
btw im very pleased to report that all the pain of this pr was in dealing with non-`ProgramTest` setups that are suddenly missing the stake program. zero issues with any of the `ProgramTest` consumers that now have a different stake program